<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QDialog" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_sec_status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>471</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Profile ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pid"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fname"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lname"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="address"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Country</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_country"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Haplogroup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_haplogroup"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>HV1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_hv1"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>HV11</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_hv2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>ATP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_atp"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QPushButton" name="pushButton_add">
          <property name="text">
           <string>ADD</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_edit">
        <property name="text">
         <string>EDIT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>DELETE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>301</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_16">
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Search By Profile Id</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Search By haplogroup</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Search By country</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Search By Last Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Search By first Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label_sec_status</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>comboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
